---
layout: page
title: Teaching
---


## Bioinformatics & Population Genomics

[Practicals](//wurmlab.github.io/genomicscourse/2016-SIB/practicals/) prepared with Rodrigo Pracana, Julien Roux, Rob Waterhouse and Stefan Schiffels for a 1 week [Bioinformatics & Population Genomics ](//www.isb-sib.ch/events/training/joint-spring-school-bioinformatics-and-population-genomics) course.  [[Overview slides]](http://www.slideshare.net/yannickwurm/2016-0529introsibspringschoolleuker-bad). We use similar material for [QMUL's Bioinformatics MSc](//www.sbcs.qmul.ac.uk/postgraduate/masters/courses/169225.htmll) and our [Ecological and Evolutionary Genomics MSc](//www.sbcs.qmul.ac.uk/postgraduate/masters/courses/169221.html):

* [Reads to genome to gene predictions](//wurmlab.github.io/genomicscourse/2016-SIB/practicals/reference_genome/assembly):<br/>
   Short read cleaning, genome assembly, quality control, gene prediction, quality control. [[Slides]](http://www.slideshare.net/yannickwurm/2016-0530mondayassembly)
* Population sequencing to genotypes to population genetics statistics:
     * [Mapping reads, calling variants, visualizing variant calls.](//wurmlab.github.io/genomicscourse/2016-SIB/practicals/population_genetics/map_call)
     * [Analysing variants: PCA, measuring Differentiation & Diversity.](//wurmlab.github.io/genomicscourse/2016-SIB/practicals/population_genetics/popgen)
* [Inferring population sizes and gene flow.](//wurmlab.github.io/genomicscourse/2016-SIB/practicals/msmc/msmc-tutorial/guide)
* Gene expression:
     * [From raw sequencing data to transcript expression levels.](//wurmlab.github.io/genomicscourse/2016-SIB/practicals/rnaseq/TP1)
     * [Gene-level clustering and differential expression analysis.](//wurmlab.github.io/genomicscourse/2016-SIB/practicals/rnaseq/TP2)



## Ecological and Evolutionary Genomics

Our students did some great blogging. 



## Learning how to program in R

<div class="row">
  <div class="span7">

    As biology is becoming a data science its increasingly important to be able to wrangle large datasets. We use variations of the following for QMUL's SBC361 (3rd year Research Methods) and Bio781p (MSc Statistics & Bioinformatics):

    <ul>
      <li><a href="//wurmlab.github.io/SBC361-programming-in-R/practical1.html">Practical 1: R refresher and regular expressions</a></li>
      <li><a href="//https://wurmlab.github.io/SBC361-programming-in-R/practical2.html">Practical 2: Functions and for loops</a></li>
      <li><a href="//wurmlab.github.io/SBC361-programming-in-R/practical3.html">Practical 3: Practice to pull it all together</a></li>
    </ul>

  </div>
  <div class="span3">
    
    <div style="left: 0px; width: 100%; height: 0px; position: relative; padding-bottom: 75.0019%; padding-top: 38px;"><iframe src="https://www.slideshare.net/slideshow/embed_code/key/jCQoRUBEZOsam0" frameborder="0" allowfullscreen="true" webkitallowfullscreen="true" mozallowfullscreen="true" style="top: 0px; left: 0px; width: 100%; height: 100%; position: absolute;"></iframe></div>
 
  </div>
</div>



## Evolution

1st year introduction (part of QMUL's BIO113):

<div style="text-align:center;" class="row">
  <div class="span3">
    <!-- 1 -->
    <div style="width: 100%; height: 0px; position: relative; padding-bottom: 75.0019%; padding-top: 38px;">
      <iframe src="https://www.slideshare.net/slideshow/embed_code/key/DxhOYKI8g76CcU" frameborder="0" allowfullscreen style="width: 100%; height: 100%; position: absolute;"></iframe>
    </div>
  </div>
  <div class="span3">
    <!-- 2 -->
    <div style="width: 100%; height: 0px; position: relative; padding-bottom: 75.0019%; padding-top: 38px;">
      <iframe src="//www.slideshare.net/slideshow/embed_code/key/6n2KuXrF13QOay" frameborder="0" allowfullscreen style="width: 100%; height: 100%; position: absolute;"> </iframe>
    </div>
  </div>
  <div class="span3">
    <!-- 3 -->
    <div style="width: 100%; height: 0px; position: relative; padding-bottom: 75.0019%; padding-top: 38px;">
      <iframe src="//www.slideshare.net/slideshow/embed_code/key/EbEcpxCEUx66x1" frameborder="0" allowfullscreen style="width: 100%; height: 100%; position: absolute;"> </iframe>
    </div>
  </div>
  <div class="span3">
    <!-- 4 -->
    <div style="width: 100%; height: 0px; position: relative; padding-bottom: 75.0019%; padding-top: 38px;">
      <iframe src="//www.slideshare.net/slideshow/embed_code/key/Y9nUUktwu7G2u" frameborder="0" allowfullscreen style="width: 100%; height: 100%; position: absolute;"> </iframe>
    </div>
  </div>
  <div class="span3">
    <!-- 5 -->
    <div style="width: 100%; height: 0px; position: relative; padding-bottom: 75.0019%; padding-top: 38px;">
      <iframe src="//www.slideshare.net/slideshow/embed_code/key/LOuf0tvzOMHvwr" frameborder="0" allowfullscreen style="width: 100%; height: 100%; position: absolute;"> </iframe>
      
    </div>
  </div>
</div>
<!-- old was: width="100%" height="390" f -->
        

<div class="row">

 <div class="span12">
<h2>Older stuff</h2>

<h3>2012 Swiss Bioinformatics Institute summerschool</h3>

From 2012 Swiss Bioinformatics Institute <a href="//www.isb-sib.ch/education/sib-phd-training-network/events/past-events/phd-summer-school-2012.html">summerschool in Bioinformatics and Population Genomics</a> (with Oksana Riba-Grognuz, Kate Ridout and Eyal Privman). This is largely superceded by the newer version (see above):

<ul>
  <li><a href="//www.slideshare.net/yannickwurm/20120622-fridayadelboden" title="20120622 fridayadelboden" target="_blank">Overview and summary slides</a></li>
  <li><a href="DAY1_exercises.docx.html">UNIX introduction, <em>de novo</em> genome assembly &amp; <em>de novo</em> transcriptome assembly, and cross-validation using Cytoscape of the genome and transcriptome assemblies</a>. [<a href="https://docs.google.com/document/pub?id=1lx8cGQlyBZV8RXm7gSLqFafcOUfPq6_GjadqBxqiAzQ">Google Docs link.</a>]</li>
  <li><a href="MWAS%20practical.docx.html">Automated gene model prediction and manual refinement</a>.[<a href="https://docs.google.com/document/pub?id=10P7oSJ3M_Jg6EzSPbsKx1HV6WQprOURcpvIBxa5Yk6g">Google Docs link.</a>]</li>
  <li><a href="DAY3_exercises_Oksana_WORKS_denovo.docx.html">Differential gene expression with tophat/cufflinks/cummerbund</a>.[<a href="https://docs.google.com/document/pub?id=1UB8I67AqW1nPbIo1JN4td76Bbyu1b1hGxnr9Yoe2t-k">Google Docs link.</a>]</li>
  <li><a href="//www.slideshare.net/yannickwurm/20120622-fridayadelboden">Some random tips</a> and <a href="Experimental%20design%20for%20genomics.html">discussion topics</a>.
[<a href="https://docs.google.com/document/pub?id=1WcHIJPzkOXXjnViwJt_ZVERlJz2YUqhM6yXQ1OLQTxQ">Google Docs link.</a>]</li>
</ul>


</div>
</div>

Introduction to bioinformatics & bioruby @ <a href="//tech.groups.yahoo.com/group/id-ruby/">Jakarta.rb</a> (2010-10-31):

<div><div class="span1" style="text-align: center;"><div style="width: 100%; height: 0px; position: relative; padding-bottom: 75.0019%; padding-top: 38px;"><iframe src="https://www.slideshare.net/slideshow/embed_code/key/d1laaAj50DPgFx" frameborder="0" allowfullscreen style="width: 100%; height: 100%; position: absolute;"></iframe></div></div></div>

<div class="row">

 <div class="span12">

<h3>2006-2010 stuff from Uni Lausanne:</h3>

<ul>
  <li><a href="bachelor3-tp4Microarray">Introduction to microarray analysis</a>: Three hour practical for 3rd year Biology students. Download normalized affimterix data, blindly use Limma to find significant genes, cluster them with TMEV, conduct GO analysis of one cluster… conclude about confounding factors in experimental design.</li>
  <li><a href="bachelor2-tp2Blast">Introduction to BLAST</a>: Two hour practical for 2nd year biology students. Calculate score &amp; evalue of an alignment by hand… compare with true Blast results obtained on NCBI, see that blastp and blastn have different sensitivities. Check some new NCBI blast features.</li>
</ul>

<p>Semester programming projects:</p>

<ul>
  <li><strong>Programming project</strong>: for introduction to python/biopython class. <a href="bachelor2-introToProg/ProgrammingForBiologists_Wurm_project.pdf">Determining Codon Adaptation Index in ants</a>. Required datafiles: <a href="bachelor2-introToProg/SI_putativeTranscripts.fasta">sequence fasta file</a> and <a href="bachelor2-introToProg/SI_All_bx">blastx report</a>.</li>
  <li><strong>Programming project</strong>: for introduction to python/biopython class. <a href="bachelor2-introToProg/ProteinCorrectionProject.pdf">Improved protein sequences for non-model organisms</a>.</li>
</ul>


</div></div>
