

You need to learn 4 things:
1. Unix  (for running software & moving files on the cluster)
2. R  (for handling numbers & doing things like RNAseq analyses)
3. Statistics & experimental design (for understanding what you are telling R to do & to understand what is a true result and what is just due to chance). 
4. Scripting in a different language (python or ruby)


Any bioinformaticsy project involves all four. While there is some overlap in that they are all abstract manners of thinking about things, at a technical level they are complementary. They also represent different forms of abstraction.


As part of this I recommend you take the following modules (from ~September 25th; they are each 2 weeks fulltime +1 week coursework): 
 - Genome Bioinformatics       BIO721P   (will help with 1 as well as general genome biology concepts)
 - Statistics & Bioinformatics BIO781P   (will help with 2 & 3)
 - Coding for Scientists       BIO723P   (will help for 4)



In the meantime, you can get a small taster of some things through the following. 

1. I recommend two things: 
   - [The swiss institute of Bioinformatics UNIX fundamentals online course](https://edu.isb-sib.ch/course/view.php?id=82)
   - [Command-line bootcamp on the web](http://rik.smith-unna.com/command_line_bootcamp/)

2. For basic datahandling in R. In the past I have used and recommended these: 
  - http://tryr.codeschool.com
  - a similar approach is to use the swirl() library from http://swirlstats.com
      * follow the "R programming" self-led course (you can skip the sections "Simulation" and "Dates and Times")
      * follow at least the ggplot section of the "Exploratory_Data_Analysis" swirl course

  But the following may be a relevant alternative that is easier to do autonomously because it also includes videos (and the website seems mature, but some parts may cost something). This also includes elements of 3.
     https://www.datacamp.com/courses/free-introduction-to-r
     https://www.datacamp.com/tracks/statistics-with-r
     https://www.datacamp.com/courses/analysis-of-variance-anova

  (with plotting in R, don't use the base R graphics which are mediocre, always use ggplot if that is an option).

4. Keep the scripting in python/ruby for later. 


If you don't yet have a macbook, you should get one. I recommend a "standard" 15 inch macbook pro (the *pro* are more powerful (and slighly heavier) than the others but at a similar price). We'll eventually get a screen for you to atttach it to here. There is not much point trying to do bioinformatics on a windows machine (and none of us will be able to help you). The alternative is to switch to Linux full-time, but that can create other challenges (e.g. word & powerpoint & excel don't exist).

Courses for this kind of stuff may exist. Check what may be available at EMBO, Wellcome Sanger, EBI, "software carpentry".
