<!DOCTYPE html>
<html>
<head>
  <title>GeneValidator: identify problems with gene predictions</title>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type">

  <script type="text/javascript" src="files/js/jquery-2.1.1.min.js"></script>
  <script type="text/javascript" src="files/js/bootstrap.min.js"></script>
  <script type="text/javascript" src="files/js/jquery.tablesorter.min.js"></script>
  <script type="text/javascript" src="files/js/d3.v3.min.js"></script>
  <script type="text/javascript" src="files/js/script.js"></script>
  <script type="text/javascript" src="files/js/plots.js"></script>

  <link href="files/css/bootstrap.min.css" rel="stylesheet" type="text/css">
  <link href="files/css/font-awesome.min.css" rel="stylesheet" type="text/css">
  <link href="files/css/style.css" rel="stylesheet" type="text/css">

</head>
<body>
  <div class="container">
    <span class="menu_icon pull-right"><a href="#" id="show_all_plots" onclick="show_all_plots(this);"><i class="fa fa-2x fa-bar-chart-o"></i><br>Show All Charts</a></span>
    <span class="menu_icon pull-right"><a href="https://github.com/monicadragan/GeneValidator" target="_blank"><i class="fa fa-2x fa-github"></i><br>Source Code</a></span>
    <span class="menu_icon pull-right"><a href="http://wurmlab.github.io/tools/genevalidator/documentation/v1/"><i class="fa fa-2x fa-info-circle"></i><br>About</a></span>
    <div class="clearfix"></div>
    <img class="logo" src="files/img/gene.png" alt="logo">
    <div class="page-title">
      <h1>Gene Validator <small>v1.5.4</small></h1>
      <h4 class="subheading">Identify Problems with Gene Predictions</h4>
    </div>
    <br/><br/>
    <div id="report">
      <div id="report_1"></div>
    </div>
    <br><br>
      <table id="sortable_table" class="table table-collapsed table-bordered table-condensed tablesorter">
        <thead>
          <tr id="header">
            <th>#</th>
            <th>Ranking</th>
            <th>Sequence Definition
              <span data-toggle="tooltip" title="Query definition as it apears in the input fasta file." data-placement="top"><i class="fa fa-question-circle"></i></span>
            </th>
            <th>No. Hits
              <span data-toggle="tooltip" title="Number of non-identical hits found by BLAST." data-placement="top"><i class="fa fa-question-circle"></i></span>
            </th>
                        <th class="sorter-false">
              <b>Length Cluster</b>
                                  <span data-toggle="tooltip" title="Charts available for this validation" data-placement="top"><i class="fa fa-bar-chart-o chartIcon"></i></span>&nbsp;<span data-toggle="tooltip" title="Check whether the prediction length fits most of the BLAST hit lengths, by 1D hierarchical clusterization. Meaning of the output displayed: Query_length [Main Cluster Length Interval]" data-placement="top"><i class="fa fa-question-circle"></i></span>
                            </th>
                        <th class="sorter-false">
              <b>Length Rank</b>
                                   <span data-toggle="tooltip" title="Check whether the rank of the prediction length lies among 80% of all the BLAST hit lengths." data-placement="top"><i class="fa fa-question-circle"></i></span>
                            </th>
                        <th class="sorter-false">
              <b>Gene Merge</b>
                                  <span data-toggle="tooltip" title="Charts available for this validation" data-placement="top"><i class="fa fa-bar-chart-o chartIcon"></i></span>&nbsp;<span data-toggle="tooltip" title="Check whether BLAST hits make evidence about a merge of two genes that match the predicted gene." data-placement="top"><i class="fa fa-question-circle"></i></span>
                            </th>
                        <th class="sorter-false">
              <b>Duplication</b>
                                   <span data-toggle="tooltip" title="Check whether there is a duplicated subsequence in the predicted gene by counting the hsp residue coverage of the prediction, for each hit." data-placement="top"><i class="fa fa-question-circle"></i></span>
                            </th>
                        <th class="sorter-false">
              <b></b>
                                   <span data-toggle="tooltip" title="" data-placement="top"><i class="fa fa-question-circle"></i></span>
                            </th>
                        <th class="sorter-false">
              <b></b>
                                   <span data-toggle="tooltip" title="" data-placement="top"><i class="fa fa-question-circle"></i></span>
                            </th>
                        <th class="sorter-false">
              <b>Missing/Extra sequences</b>
                                  <span data-toggle="tooltip" title="Charts available for this validation" data-placement="top"><i class="fa fa-bar-chart-o chartIcon"></i></span>&nbsp;<span data-toggle="tooltip" title="Finds missing and extra sequences in the prediction, based on the multiple alignment of the best hits. Also counts the percentage of the conserved regions that appear in the prediction." data-placement="top"><i class="fa fa-question-circle"></i></span>
                            </th>
                        <th class="sorter-false chart-column"></th>
          </tr>
        </thead>
        <tbody><tr>
  <td>1</td>
  <td data-score="100"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:85.0%;"></div></div></td>
  <td title="Definition" class="seq_definition">GB10034-PA  (amel_OGSv1.0_pep.fa_34)</td>
  <td title="No. Hits">500</td>
      <td title="Length Cluster" class="success my-btn-success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 358 to 842 amino-acids long. The query sequence is 554 amino-acids long and is thus inside the most dense length-cluster of BLAST hits." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle1');
          addPlot('toggle1', 'files/json/protein_data.fasta_1_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '554', '');
    AddExplanation(this, 'toggle1');">554&nbsp;[358,&nbsp;842]</td>
        <td title="Length Rank" class="success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  554 amino-acids long. BLAST identified 500 hit sequences with lengths from 53 to 2265 amino-acids (median: 680; mean: 685). 147 of these hit sequences (i.e. 29%) are longer than the query sequence." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle1'); AddExplanation(this,'toggle1');">29%</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The distribution of start and/or end-coordinates of HSPs are multi-modal. To detect potential problems we performed a linear regression (with coordinates weighted inversely proportionally to  hit strength). The resulting slope is -0.2." data-conclusion="This slope is  outside our empirically calculated thresholds (0.4 and 1.2). There is no evidence that the query contains sequence from multiple genes." onclick="showDiv(this, 'toggle1');
          addPlot('toggle1', 'files/json/protein_data.fasta_1_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '526.2014950326745', '-0.19717443640097448');
          addPlot('toggle1', 'files/json/protein_data.fasta_1_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');
    AddExplanation(this, 'toggle1');">-0.2</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle1'); AddExplanation(this,'toggle1');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 100% amino-acid residues present in the consensus model. 2% of residues in the query sequence are absent from the consensus profile.  0% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle1');
          addPlot('toggle1', 'files/json/protein_data.fasta_1_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle1');">100%&nbsp;conserved; 2%&nbsp;extra; 0%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle1');
                   addPlot('toggle1', 'files/json/protein_data.fasta_1_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '554', '');                                   addPlot('toggle1', 'files/json/protein_data.fasta_1_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '526.2014950326745', '-0.19717443640097448'); addPlot('toggle1', 'files/json/protein_data.fasta_1_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');                                                       addPlot('toggle1', 'files/json/protein_data.fasta_1_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle1row">
    <div id="toggle1" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>10</td>
  <td data-score="0"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:0.0%;"></div></div></td>
  <td title="Definition" class="seq_definition">sp|P83011|13KDA_SCYCA  13.2 kDa protein (Fragment) OS=Scyliorhinus canicula PE=1 SV=1</td>
  <td title="No. Hits">0</td>
      <td title="Length Cluster" class="warning">Not enough evidence</td>
        <td title="Length Rank" class="warning">Not enough evidence</td>
        <td title="Gene Merge" class="warning">Not enough evidence</td>
        <td title="Duplication" class="warning">Not enough evidence</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="warning">Not enough evidence</td>
  
<td></td>
</tr>
<tr class="tablesorter-childRow" style="display:none;">
  <td colspan="14">
    <div></div>
  </td>
</tr>
<tr>
  <td>6</td>
  <td data-score="75"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:63.75%;"></div></div></td>
  <td title="Definition" class="seq_definition">GB10071-PA </td>
  <td title="No. Hits">500</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 412 to 472 amino-acids long. The query sequence is 494 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too long." onclick="showDiv(this, 'toggle6');
          addPlot('toggle6', 'files/json/protein_data.fasta_6_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '494', '');
    AddExplanation(this, 'toggle6');">494&nbsp;[412,&nbsp;472]</td>
        <td title="Length Rank" class="danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  494 amino-acids long. BLAST identified 500 hit sequences with lengths from 128 to 1647 amino-acids (median: 416; mean: 405). 34 of these hit sequences (i.e. 7%) are shorter than the query sequence." data-conclusion="The sequence may be too long." onclick="showDiv(this, 'toggle6'); AddExplanation(this,'toggle6');">7%&nbsp;(too&nbsp;long)</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The start coordinates and the end coordinates of HSPs are unimodally distributed." data-conclusion="This suggest that the query sequence represents a single gene." onclick="showDiv(this, 'toggle6');
          addPlot('toggle6', 'files/json/protein_data.fasta_6_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', '');
          addPlot('toggle6', 'files/json/protein_data.fasta_6_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');
    AddExplanation(this, 'toggle6');">0.0</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle6'); AddExplanation(this,'toggle6');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 100% amino-acid residues present in the consensus model. 0% of residues in the query sequence are absent from the consensus profile.  2% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle6');
          addPlot('toggle6', 'files/json/protein_data.fasta_6_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle6');">100%&nbsp;conserved; 0%&nbsp;extra; 2%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle6');
                   addPlot('toggle6', 'files/json/protein_data.fasta_6_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '494', '');                                   addPlot('toggle6', 'files/json/protein_data.fasta_6_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', ''); addPlot('toggle6', 'files/json/protein_data.fasta_6_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');                                                       addPlot('toggle6', 'files/json/protein_data.fasta_6_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle6row">
    <div id="toggle6" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>15</td>
  <td data-score="71"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:60.35%;"></div></div></td>
  <td title="Definition" class="seq_definition">PB26730-RA  protein AED:1 QI:0|0|0|0|0|0|10|0|347</td>
  <td title="No. Hits">351</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 388 to 422 amino-acids long. The query sequence is 347 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too short." onclick="showDiv(this, 'toggle15');
          addPlot('toggle15', 'files/json/protein_data.fasta_15_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '347', '');
    AddExplanation(this, 'toggle15');">347&nbsp;[388,&nbsp;422]</td>
        <td title="Length Rank" class="success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  347 amino-acids long. BLAST identified 351 hit sequences with lengths from 73 to 1706 amino-acids (median: 390; mean: 356). 89 of these hit sequences (i.e. 25%) are longer than the query sequence." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle15'); AddExplanation(this,'toggle15');">25%</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The distribution of start and/or end-coordinates of HSPs are multi-modal. To detect potential problems we performed a linear regression (with coordinates weighted inversely proportionally to  hit strength). The resulting slope is 0.3." data-conclusion="This slope is  outside our empirically calculated thresholds (0.4 and 1.2). There is no evidence that the query contains sequence from multiple genes." onclick="showDiv(this, 'toggle15');
          addPlot('toggle15', 'files/json/protein_data.fasta_15_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '159.50515766378277', '0.27088793044740395');
          addPlot('toggle15', 'files/json/protein_data.fasta_15_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '351', '');
    AddExplanation(this, 'toggle15');">0.3</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle15'); AddExplanation(this,'toggle15');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="danger my-btn-danger" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 100% amino-acid residues present in the consensus model. 20% of residues in the query sequence are absent from the consensus profile.  16% of residues in the consensus profile are absent from the query sequence." data-conclusion="These results suggest that there may be some problems with the query sequence." onclick="showDiv(this, 'toggle15');
          addPlot('toggle15', 'files/json/protein_data.fasta_15_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle15');">100%&nbsp;conserved; 20%&nbsp;extra; 16%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle15');
                   addPlot('toggle15', 'files/json/protein_data.fasta_15_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '347', '');                                   addPlot('toggle15', 'files/json/protein_data.fasta_15_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '159.50515766378277', '0.27088793044740395'); addPlot('toggle15', 'files/json/protein_data.fasta_15_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '351', '');                                                       addPlot('toggle15', 'files/json/protein_data.fasta_15_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle15row">
    <div id="toggle15" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>9</td>
  <td data-score="100"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:85.0%;"></div></div></td>
  <td title="Definition" class="seq_definition">GB10113-PA </td>
  <td title="No. Hits">500</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 220 to 546 amino-acids long. The query sequence is 659 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too long." onclick="showDiv(this, 'toggle9');
          addPlot('toggle9', 'files/json/protein_data.fasta_9_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '659', '');
    AddExplanation(this, 'toggle9');">659&nbsp;[220,&nbsp;546]</td>
        <td title="Length Rank" class="success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  659 amino-acids long. BLAST identified 500 hit sequences with lengths from 51 to 1108 amino-acids (median: 503; mean: 509). 111 of these hit sequences (i.e. 22%) are shorter than the query sequence." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle9'); AddExplanation(this,'toggle9');">22%</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The start coordinates and the end coordinates of HSPs are unimodally distributed." data-conclusion="This suggest that the query sequence represents a single gene." onclick="showDiv(this, 'toggle9');
          addPlot('toggle9', 'files/json/protein_data.fasta_9_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', '');
          addPlot('toggle9', 'files/json/protein_data.fasta_9_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');
    AddExplanation(this, 'toggle9');">0.0</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle9'); AddExplanation(this,'toggle9');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 95% amino-acid residues present in the consensus model. 7% of residues in the query sequence are absent from the consensus profile.  13% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle9');
          addPlot('toggle9', 'files/json/protein_data.fasta_9_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle9');">95%&nbsp;conserved; 7%&nbsp;extra; 13%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle9');
                   addPlot('toggle9', 'files/json/protein_data.fasta_9_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '659', '');                                   addPlot('toggle9', 'files/json/protein_data.fasta_9_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', ''); addPlot('toggle9', 'files/json/protein_data.fasta_9_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');                                                       addPlot('toggle9', 'files/json/protein_data.fasta_9_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle9row">
    <div id="toggle9" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>18</td>
  <td data-score="75"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:63.75%;"></div></div></td>
  <td title="Definition" class="seq_definition">Pbar_fem-RF  protein AED:1 QI:325|0|0|0|0|0|10|42|452</td>
  <td title="No. Hits">388</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 163 to 196 amino-acids long. The query sequence is 452 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too long." onclick="showDiv(this, 'toggle18');
          addPlot('toggle18', 'files/json/protein_data.fasta_18_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '452', '');
    AddExplanation(this, 'toggle18');">452&nbsp;[163,&nbsp;196]</td>
        <td title="Length Rank" class="danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  452 amino-acids long. BLAST identified 388 hit sequences with lengths from 64 to 527 amino-acids (median: 187; mean: 263). 16 of these hit sequences (i.e. 4%) are shorter than the query sequence." data-conclusion="The sequence may be too long." onclick="showDiv(this, 'toggle18'); AddExplanation(this,'toggle18');">4%&nbsp;(too&nbsp;long)</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The distribution of start and/or end-coordinates of HSPs are multi-modal. To detect potential problems we performed a linear regression (with coordinates weighted inversely proportionally to  hit strength). The resulting slope is 0.0." data-conclusion="This slope is  outside our empirically calculated thresholds (0.4 and 1.2). There is no evidence that the query contains sequence from multiple genes." onclick="showDiv(this, 'toggle18');
          addPlot('toggle18', 'files/json/protein_data.fasta_18_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '430.6511933108736', '0.02681409123879634');
          addPlot('toggle18', 'files/json/protein_data.fasta_18_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '388', '');
    AddExplanation(this, 'toggle18');">0.0</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle18'); AddExplanation(this,'toggle18');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 100% amino-acid residues present in the consensus model. 4% of residues in the query sequence are absent from the consensus profile.  3% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle18');
          addPlot('toggle18', 'files/json/protein_data.fasta_18_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle18');">100%&nbsp;conserved; 4%&nbsp;extra; 3%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle18');
                   addPlot('toggle18', 'files/json/protein_data.fasta_18_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '452', '');                                   addPlot('toggle18', 'files/json/protein_data.fasta_18_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '430.6511933108736', '0.02681409123879634'); addPlot('toggle18', 'files/json/protein_data.fasta_18_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '388', '');                                                       addPlot('toggle18', 'files/json/protein_data.fasta_18_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle18row">
    <div id="toggle18" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>13</td>
  <td data-score="75"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:63.75%;"></div></div></td>
  <td title="Definition" class="seq_definition">SI2.2.0_07054  locus=Si_gnF.scaffold08475[8186..10205].pep_2 quality=100.00</td>
  <td title="No. Hits">109</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 107 to 492 amino-acids long. The query sequence is 640 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too long." onclick="showDiv(this, 'toggle13');
          addPlot('toggle13', 'files/json/protein_data.fasta_13_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '640', '');
    AddExplanation(this, 'toggle13');">640&nbsp;[107,&nbsp;492]</td>
        <td title="Length Rank" class="danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  640 amino-acids long. BLAST identified 109 hit sequences with lengths from 107 to 1008 amino-acids (median: 347; mean: 407). 13 of these hit sequences (i.e. 12%) are shorter than the query sequence." data-conclusion="The sequence may be too long." onclick="showDiv(this, 'toggle13'); AddExplanation(this,'toggle13');">12%&nbsp;(too&nbsp;long)</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The distribution of start and/or end-coordinates of HSPs are multi-modal. To detect potential problems we performed a linear regression (with coordinates weighted inversely proportionally to  hit strength). The resulting slope is 0.3." data-conclusion="This slope is  outside our empirically calculated thresholds (0.4 and 1.2). There is no evidence that the query contains sequence from multiple genes." onclick="showDiv(this, 'toggle13');
          addPlot('toggle13', 'files/json/protein_data.fasta_13_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '442.69129993383507', '0.3432813492395522');
          addPlot('toggle13', 'files/json/protein_data.fasta_13_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '109', '');
    AddExplanation(this, 'toggle13');">0.3</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle13'); AddExplanation(this,'toggle13');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 100% amino-acid residues present in the consensus model. 2% of residues in the query sequence are absent from the consensus profile.  16% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle13');
          addPlot('toggle13', 'files/json/protein_data.fasta_13_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle13');">100%&nbsp;conserved; 2%&nbsp;extra; 16%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle13');
                   addPlot('toggle13', 'files/json/protein_data.fasta_13_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '640', '');                                   addPlot('toggle13', 'files/json/protein_data.fasta_13_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '442.69129993383507', '0.3432813492395522'); addPlot('toggle13', 'files/json/protein_data.fasta_13_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '109', '');                                                       addPlot('toggle13', 'files/json/protein_data.fasta_13_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle13row">
    <div id="toggle13" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>25</td>
  <td data-score="100"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:85.0%;"></div></div></td>
  <td title="Definition" class="seq_definition">PB18768-RA  protein AED:1 QI:187|0|0|0|0|0|2|127|132</td>
  <td title="No. Hits">442</td>
      <td title="Length Cluster" class="success my-btn-success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 104 to 175 amino-acids long. The query sequence is 132 amino-acids long and is thus inside the most dense length-cluster of BLAST hits." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle25');
          addPlot('toggle25', 'files/json/protein_data.fasta_25_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '132', '');
    AddExplanation(this, 'toggle25');">132&nbsp;[104,&nbsp;175]</td>
        <td title="Length Rank" class="success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  132 amino-acids long. BLAST identified 442 hit sequences with lengths from 39 to 2192 amino-acids (median: 126; mean: 135). 161 of these hit sequences (i.e. 36%) are shorter than the query sequence." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle25'); AddExplanation(this,'toggle25');">36%</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The start coordinates and the end coordinates of HSPs are unimodally distributed." data-conclusion="This suggest that the query sequence represents a single gene." onclick="showDiv(this, 'toggle25');
          addPlot('toggle25', 'files/json/protein_data.fasta_25_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', '');
          addPlot('toggle25', 'files/json/protein_data.fasta_25_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '442', '');
    AddExplanation(this, 'toggle25');">0.0</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle25'); AddExplanation(this,'toggle25');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 100% amino-acid residues present in the consensus model. 6% of residues in the query sequence are absent from the consensus profile.  0% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle25');
          addPlot('toggle25', 'files/json/protein_data.fasta_25_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle25');">100%&nbsp;conserved; 6%&nbsp;extra; 0%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle25');
                   addPlot('toggle25', 'files/json/protein_data.fasta_25_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '132', '');                                   addPlot('toggle25', 'files/json/protein_data.fasta_25_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', ''); addPlot('toggle25', 'files/json/protein_data.fasta_25_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '442', '');                                                       addPlot('toggle25', 'files/json/protein_data.fasta_25_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle25row">
    <div id="toggle25" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>2</td>
  <td data-score="100"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:85.0%;"></div></div></td>
  <td title="Definition" class="seq_definition">GB10040-PA  (amel_OGSv1.0_pep.fa_40)</td>
  <td title="No. Hits">500</td>
      <td title="Length Cluster" class="success my-btn-success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 650 to 1073 amino-acids long. The query sequence is 1070 amino-acids long and is thus inside the most dense length-cluster of BLAST hits." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle2');
          addPlot('toggle2', 'files/json/protein_data.fasta_2_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '1070', '');
    AddExplanation(this, 'toggle2');">1070&nbsp;[650,&nbsp;1073]</td>
        <td title="Length Rank" class="success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  1070 amino-acids long. BLAST identified 500 hit sequences with lengths from 281 to 2120 amino-acids (median: 967; mean: 994). 213 of these hit sequences (i.e. 43%) are shorter than the query sequence." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle2'); AddExplanation(this,'toggle2');">43%</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The start coordinates and the end coordinates of HSPs are unimodally distributed." data-conclusion="This suggest that the query sequence represents a single gene." onclick="showDiv(this, 'toggle2');
          addPlot('toggle2', 'files/json/protein_data.fasta_2_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', '');
          addPlot('toggle2', 'files/json/protein_data.fasta_2_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');
    AddExplanation(this, 'toggle2');">0.0</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle2'); AddExplanation(this,'toggle2');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 100% amino-acid residues present in the consensus model. 0% of residues in the query sequence are absent from the consensus profile.  7% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle2');
          addPlot('toggle2', 'files/json/protein_data.fasta_2_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle2');">100%&nbsp;conserved; 0%&nbsp;extra; 7%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle2');
                   addPlot('toggle2', 'files/json/protein_data.fasta_2_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '1070', '');                                   addPlot('toggle2', 'files/json/protein_data.fasta_2_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', ''); addPlot('toggle2', 'files/json/protein_data.fasta_2_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');                                                       addPlot('toggle2', 'files/json/protein_data.fasta_2_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle2row">
    <div id="toggle2" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>8</td>
  <td data-score="71"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:60.35%;"></div></div></td>
  <td title="Definition" class="seq_definition">GB10098-PA </td>
  <td title="No. Hits">500</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 1030 to 1093 amino-acids long. The query sequence is 826 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too short." onclick="showDiv(this, 'toggle8');
          addPlot('toggle8', 'files/json/protein_data.fasta_8_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '826', '');
    AddExplanation(this, 'toggle8');">826&nbsp;[1030,&nbsp;1093]</td>
        <td title="Length Rank" class="success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  826 amino-acids long. BLAST identified 500 hit sequences with lengths from 310 to 1408 amino-acids (median: 1050; mean: 973). 117 of these hit sequences (i.e. 23%) are longer than the query sequence." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle8'); AddExplanation(this,'toggle8');">23%</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The start coordinates and the end coordinates of HSPs are unimodally distributed." data-conclusion="This suggest that the query sequence represents a single gene." onclick="showDiv(this, 'toggle8');
          addPlot('toggle8', 'files/json/protein_data.fasta_8_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', '');
          addPlot('toggle8', 'files/json/protein_data.fasta_8_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');
    AddExplanation(this, 'toggle8');">0.0</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle8'); AddExplanation(this,'toggle8');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="danger my-btn-danger" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 78% amino-acid residues present in the consensus model. 0% of residues in the query sequence are absent from the consensus profile.  21% of residues in the consensus profile are absent from the query sequence." data-conclusion="These results suggest that there may be some problems with the query sequence. The query sequence has a high percentage (21%) of missing residues when compared to the statistical profile (the cut-off is 20%)." onclick="showDiv(this, 'toggle8');
          addPlot('toggle8', 'files/json/protein_data.fasta_8_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle8');">78%&nbsp;conserved; 0%&nbsp;extra; 21%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle8');
                   addPlot('toggle8', 'files/json/protein_data.fasta_8_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '826', '');                                   addPlot('toggle8', 'files/json/protein_data.fasta_8_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', ''); addPlot('toggle8', 'files/json/protein_data.fasta_8_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');                                                       addPlot('toggle8', 'files/json/protein_data.fasta_8_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle8row">
    <div id="toggle8" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>17</td>
  <td data-score="100"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:85.0%;"></div></div></td>
  <td title="Definition" class="seq_definition">PB18762-RA  protein AED:1 QI:0|0|0|0|0|0|2|0|207</td>
  <td title="No. Hits">500</td>
      <td title="Length Cluster" class="success my-btn-success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 171 to 388 amino-acids long. The query sequence is 207 amino-acids long and is thus inside the most dense length-cluster of BLAST hits." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle17');
          addPlot('toggle17', 'files/json/protein_data.fasta_17_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '207', '');
    AddExplanation(this, 'toggle17');">207&nbsp;[171,&nbsp;388]</td>
        <td title="Length Rank" class="danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  207 amino-acids long. BLAST identified 500 hit sequences with lengths from 113 to 584 amino-acids (median: 313; mean: 331). 21 of these hit sequences (i.e. 4%) are longer than the query sequence." data-conclusion="The sequence may be too short." onclick="showDiv(this, 'toggle17'); AddExplanation(this,'toggle17');">4%&nbsp;(too&nbsp;short)</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The start coordinates and the end coordinates of HSPs are unimodally distributed." data-conclusion="This suggest that the query sequence represents a single gene." onclick="showDiv(this, 'toggle17');
          addPlot('toggle17', 'files/json/protein_data.fasta_17_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', '');
          addPlot('toggle17', 'files/json/protein_data.fasta_17_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');
    AddExplanation(this, 'toggle17');">0.0</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle17'); AddExplanation(this,'toggle17');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 83% amino-acid residues present in the consensus model. 0% of residues in the query sequence are absent from the consensus profile.  17% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle17');
          addPlot('toggle17', 'files/json/protein_data.fasta_17_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle17');">83%&nbsp;conserved; 0%&nbsp;extra; 17%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle17');
                   addPlot('toggle17', 'files/json/protein_data.fasta_17_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '207', '');                                   addPlot('toggle17', 'files/json/protein_data.fasta_17_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', ''); addPlot('toggle17', 'files/json/protein_data.fasta_17_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');                                                       addPlot('toggle17', 'files/json/protein_data.fasta_17_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle17row">
    <div id="toggle17" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>11</td>
  <td data-score="75"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:63.75%;"></div></div></td>
  <td title="Definition" class="seq_definition">sp|Q8GBW6|12S_PROFR  Methylmalonyl-CoA carboxyltransferase 12S subunit OS=Propionibacterium freudenreichii subsp. shermanii PE=1 SV=3</td>
  <td title="No. Hits">499</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 511 to 518 amino-acids long. The query sequence is 611 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too long." onclick="showDiv(this, 'toggle11');
          addPlot('toggle11', 'files/json/protein_data.fasta_11_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '611', '');
    AddExplanation(this, 'toggle11');">611&nbsp;[511,&nbsp;518]</td>
        <td title="Length Rank" class="danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  611 amino-acids long. BLAST identified 499 hit sequences with lengths from 438 to 605 amino-acids (median: 516; mean: 516). All hit sequences are shorter than the query sequence." data-conclusion="The sequence may be too long." onclick="showDiv(this, 'toggle11'); AddExplanation(this,'toggle11');">0%&nbsp;(too&nbsp;long)</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The distribution of start and/or end-coordinates of HSPs are multi-modal. To detect potential problems we performed a linear regression (with coordinates weighted inversely proportionally to  hit strength). The resulting slope is -0.6." data-conclusion="This slope is  outside our empirically calculated thresholds (0.4 and 1.2). There is no evidence that the query contains sequence from multiple genes." onclick="showDiv(this, 'toggle11');
          addPlot('toggle11', 'files/json/protein_data.fasta_11_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '532.2487757712548', '-0.6202364705632991');
          addPlot('toggle11', 'files/json/protein_data.fasta_11_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '499', '');
    AddExplanation(this, 'toggle11');">-0.6</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle11'); AddExplanation(this,'toggle11');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 100% amino-acid residues present in the consensus model. 14% of residues in the query sequence are absent from the consensus profile.  0% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle11');
          addPlot('toggle11', 'files/json/protein_data.fasta_11_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle11');">100%&nbsp;conserved; 14%&nbsp;extra; 0%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle11');
                   addPlot('toggle11', 'files/json/protein_data.fasta_11_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '611', '');                                   addPlot('toggle11', 'files/json/protein_data.fasta_11_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '532.2487757712548', '-0.6202364705632991'); addPlot('toggle11', 'files/json/protein_data.fasta_11_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '499', '');                                                       addPlot('toggle11', 'files/json/protein_data.fasta_11_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle11row">
    <div id="toggle11" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>19</td>
  <td data-score="50"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:42.5%;"></div></div></td>
  <td title="Definition" class="seq_definition">PB18775-RA  protein AED:1 QI:0|0|0|0|0|0|17|0|1160</td>
  <td title="No. Hits">500</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 613 to 722 amino-acids long. The query sequence is 1160 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too long." onclick="showDiv(this, 'toggle19');
          addPlot('toggle19', 'files/json/protein_data.fasta_19_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '1160', '');
    AddExplanation(this, 'toggle19');">1160&nbsp;[613,&nbsp;722]</td>
        <td title="Length Rank" class="danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  1160 amino-acids long. BLAST identified 500 hit sequences with lengths from 313 to 1093 amino-acids (median: 656; mean: 611). All hit sequences are shorter than the query sequence." data-conclusion="The sequence may be too long." onclick="showDiv(this, 'toggle19'); AddExplanation(this,'toggle19');">0%&nbsp;(too&nbsp;long)</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The distribution of start and/or end-coordinates of HSPs are multi-modal. To detect potential problems we performed a linear regression (with coordinates weighted inversely proportionally to  hit strength). The resulting slope is 0.0." data-conclusion="This slope is  outside our empirically calculated thresholds (0.4 and 1.2). There is no evidence that the query contains sequence from multiple genes." onclick="showDiv(this, 'toggle19');
          addPlot('toggle19', 'files/json/protein_data.fasta_19_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '1096.9102270409421', '0.04298803459301496');
          addPlot('toggle19', 'files/json/protein_data.fasta_19_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');
    AddExplanation(this, 'toggle19');">0.0</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle19'); AddExplanation(this,'toggle19');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="danger my-btn-danger" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 95% amino-acid residues present in the consensus model. 38% of residues in the query sequence are absent from the consensus profile.  8% of residues in the consensus profile are absent from the query sequence." data-conclusion="These results suggest that there may be some problems with the query sequence. The query sequence has a high percentage (38%) of extra residues absent from the statistical profile (the cut-off is 20%)." onclick="showDiv(this, 'toggle19');
          addPlot('toggle19', 'files/json/protein_data.fasta_19_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle19');">95%&nbsp;conserved; 38%&nbsp;extra; 8%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle19');
                   addPlot('toggle19', 'files/json/protein_data.fasta_19_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '1160', '');                                   addPlot('toggle19', 'files/json/protein_data.fasta_19_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '1096.9102270409421', '0.04298803459301496'); addPlot('toggle19', 'files/json/protein_data.fasta_19_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');                                                       addPlot('toggle19', 'files/json/protein_data.fasta_19_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle19row">
    <div id="toggle19" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>4</td>
  <td data-score="100"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:85.0%;"></div></div></td>
  <td title="Definition" class="seq_definition">GB10058-PA </td>
  <td title="No. Hits">500</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 1109 to 1359 amino-acids long. The query sequence is 1052 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too short." onclick="showDiv(this, 'toggle4');
          addPlot('toggle4', 'files/json/protein_data.fasta_4_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '1052', '');
    AddExplanation(this, 'toggle4');">1052&nbsp;[1109,&nbsp;1359]</td>
        <td title="Length Rank" class="success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  1052 amino-acids long. BLAST identified 500 hit sequences with lengths from 311 to 1583 amino-acids (median: 1154; mean: 1098). 176 of these hit sequences (i.e. 35%) are longer than the query sequence." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle4'); AddExplanation(this,'toggle4');">35%</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The distribution of start and/or end-coordinates of HSPs are multi-modal. To detect potential problems we performed a linear regression (with coordinates weighted inversely proportionally to  hit strength). The resulting slope is 0.0." data-conclusion="This slope is  outside our empirically calculated thresholds (0.4 and 1.2). There is no evidence that the query contains sequence from multiple genes." onclick="showDiv(this, 'toggle4');
          addPlot('toggle4', 'files/json/protein_data.fasta_4_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '1037.5061085153854', '-0.03347441034892182');
          addPlot('toggle4', 'files/json/protein_data.fasta_4_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');
    AddExplanation(this, 'toggle4');">0.0</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle4'); AddExplanation(this,'toggle4');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 95% amino-acid residues present in the consensus model. 0% of residues in the query sequence are absent from the consensus profile.  7% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle4');
          addPlot('toggle4', 'files/json/protein_data.fasta_4_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle4');">95%&nbsp;conserved; 0%&nbsp;extra; 7%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle4');
                   addPlot('toggle4', 'files/json/protein_data.fasta_4_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '1052', '');                                   addPlot('toggle4', 'files/json/protein_data.fasta_4_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '1037.5061085153854', '-0.03347441034892182'); addPlot('toggle4', 'files/json/protein_data.fasta_4_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');                                                       addPlot('toggle4', 'files/json/protein_data.fasta_4_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle4row">
    <div id="toggle4" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>22</td>
  <td data-score="75"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:63.75%;"></div></div></td>
  <td title="Definition" class="seq_definition">PB13758-RA  protein AED:1 QI:0|0|0|0|0|0|14|0|751</td>
  <td title="No. Hits">500</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 829 to 855 amino-acids long. The query sequence is 751 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too short." onclick="showDiv(this, 'toggle22');
          addPlot('toggle22', 'files/json/protein_data.fasta_22_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '751', '');
    AddExplanation(this, 'toggle22');">751&nbsp;[829,&nbsp;855]</td>
        <td title="Length Rank" class="danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  751 amino-acids long. BLAST identified 500 hit sequences with lengths from 486 to 1196 amino-acids (median: 841; mean: 827). 37 of these hit sequences (i.e. 7%) are longer than the query sequence." data-conclusion="The sequence may be too short." onclick="showDiv(this, 'toggle22'); AddExplanation(this,'toggle22');">7%&nbsp;(too&nbsp;short)</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The start coordinates and the end coordinates of HSPs are unimodally distributed." data-conclusion="This suggest that the query sequence represents a single gene." onclick="showDiv(this, 'toggle22');
          addPlot('toggle22', 'files/json/protein_data.fasta_22_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', '');
          addPlot('toggle22', 'files/json/protein_data.fasta_22_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');
    AddExplanation(this, 'toggle22');">0.0</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle22'); AddExplanation(this,'toggle22');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 91% amino-acid residues present in the consensus model. 0% of residues in the query sequence are absent from the consensus profile.  17% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle22');
          addPlot('toggle22', 'files/json/protein_data.fasta_22_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle22');">91%&nbsp;conserved; 0%&nbsp;extra; 17%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle22');
                   addPlot('toggle22', 'files/json/protein_data.fasta_22_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '751', '');                                   addPlot('toggle22', 'files/json/protein_data.fasta_22_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', ''); addPlot('toggle22', 'files/json/protein_data.fasta_22_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');                                                       addPlot('toggle22', 'files/json/protein_data.fasta_22_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle22row">
    <div id="toggle22" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>24</td>
  <td data-score="75"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:63.75%;"></div></div></td>
  <td title="Definition" class="seq_definition">PB18752-RA  protein AED:1 QI:0|0|0|0|0|0|13|0|794</td>
  <td title="No. Hits">500</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 523 to 756 amino-acids long. The query sequence is 794 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too long." onclick="showDiv(this, 'toggle24');
          addPlot('toggle24', 'files/json/protein_data.fasta_24_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '794', '');
    AddExplanation(this, 'toggle24');">794&nbsp;[523,&nbsp;756]</td>
        <td title="Length Rank" class="danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  794 amino-acids long. BLAST identified 500 hit sequences with lengths from 215 to 856 amino-acids (median: 652; mean: 649). 2 of these hit sequences (i.e. 0%) are shorter than the query sequence." data-conclusion="The sequence may be too long." onclick="showDiv(this, 'toggle24'); AddExplanation(this,'toggle24');">0%&nbsp;(too&nbsp;long)</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The start coordinates and the end coordinates of HSPs are unimodally distributed." data-conclusion="This suggest that the query sequence represents a single gene." onclick="showDiv(this, 'toggle24');
          addPlot('toggle24', 'files/json/protein_data.fasta_24_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', '');
          addPlot('toggle24', 'files/json/protein_data.fasta_24_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');
    AddExplanation(this, 'toggle24');">0.0</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle24'); AddExplanation(this,'toggle24');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 97% amino-acid residues present in the consensus model. 9% of residues in the query sequence are absent from the consensus profile.  0% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle24');
          addPlot('toggle24', 'files/json/protein_data.fasta_24_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle24');">97%&nbsp;conserved; 9%&nbsp;extra; 0%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle24');
                   addPlot('toggle24', 'files/json/protein_data.fasta_24_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '794', '');                                   addPlot('toggle24', 'files/json/protein_data.fasta_24_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', ''); addPlot('toggle24', 'files/json/protein_data.fasta_24_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');                                                       addPlot('toggle24', 'files/json/protein_data.fasta_24_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle24row">
    <div id="toggle24" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>7</td>
  <td data-score="100"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:85.0%;"></div></div></td>
  <td title="Definition" class="seq_definition">GB10091-PA </td>
  <td title="No. Hits">497</td>
      <td title="Length Cluster" class="success my-btn-success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 68 to 1090 amino-acids long. The query sequence is 708 amino-acids long and is thus inside the most dense length-cluster of BLAST hits." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle7');
          addPlot('toggle7', 'files/json/protein_data.fasta_7_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '708', '');
    AddExplanation(this, 'toggle7');">708&nbsp;[68,&nbsp;1090]</td>
        <td title="Length Rank" class="success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  708 amino-acids long. BLAST identified 497 hit sequences with lengths from 68 to 22949 amino-acids (median: 713; mean: 2102). 241 of these hit sequences (i.e. 48%) are longer than the query sequence." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle7'); AddExplanation(this,'toggle7');">48%</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The distribution of start and/or end-coordinates of HSPs are multi-modal. To detect potential problems we performed a linear regression (with coordinates weighted inversely proportionally to  hit strength). The resulting slope is -0.2." data-conclusion="This slope is  outside our empirically calculated thresholds (0.4 and 1.2). There is no evidence that the query contains sequence from multiple genes." onclick="showDiv(this, 'toggle7');
          addPlot('toggle7', 'files/json/protein_data.fasta_7_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '682.8172713732652', '-0.2080064217385302');
          addPlot('toggle7', 'files/json/protein_data.fasta_7_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '497', '');
    AddExplanation(this, 'toggle7');">-0.2</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle7'); AddExplanation(this,'toggle7');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 100% amino-acid residues present in the consensus model. 0% of residues in the query sequence are absent from the consensus profile.  1% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle7');
          addPlot('toggle7', 'files/json/protein_data.fasta_7_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle7');">100%&nbsp;conserved; 0%&nbsp;extra; 1%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle7');
                   addPlot('toggle7', 'files/json/protein_data.fasta_7_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '708', '');                                   addPlot('toggle7', 'files/json/protein_data.fasta_7_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '682.8172713732652', '-0.2080064217385302'); addPlot('toggle7', 'files/json/protein_data.fasta_7_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '497', '');                                                       addPlot('toggle7', 'files/json/protein_data.fasta_7_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle7row">
    <div id="toggle7" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>14</td>
  <td data-score="75"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:63.75%;"></div></div></td>
  <td title="Definition" class="seq_definition">SI2.2.0_02651  locus=Si_gnF.scaffold01722[932834..938174].pep_2 quality=100.00 tag=gw_corrected</td>
  <td title="No. Hits">500</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 827 to 1105 amino-acids long. The query sequence is 660 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too short." onclick="showDiv(this, 'toggle14');
          addPlot('toggle14', 'files/json/protein_data.fasta_14_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '660', '');
    AddExplanation(this, 'toggle14');">660&nbsp;[827,&nbsp;1105]</td>
        <td title="Length Rank" class="danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  660 amino-acids long. BLAST identified 500 hit sequences with lengths from 48 to 2385 amino-acids (median: 919; mean: 909). 92 of these hit sequences (i.e. 18%) are longer than the query sequence." data-conclusion="The sequence may be too short." onclick="showDiv(this, 'toggle14'); AddExplanation(this,'toggle14');">18%&nbsp;(too&nbsp;short)</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The distribution of start and/or end-coordinates of HSPs are multi-modal. To detect potential problems we performed a linear regression (with coordinates weighted inversely proportionally to  hit strength). The resulting slope is -0.5." data-conclusion="This slope is  outside our empirically calculated thresholds (0.4 and 1.2). There is no evidence that the query contains sequence from multiple genes." onclick="showDiv(this, 'toggle14');
          addPlot('toggle14', 'files/json/protein_data.fasta_14_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '622.7250344931327', '-0.45170261274150075');
          addPlot('toggle14', 'files/json/protein_data.fasta_14_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');
    AddExplanation(this, 'toggle14');">-0.5</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle14'); AddExplanation(this,'toggle14');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 94% amino-acid residues present in the consensus model. 4% of residues in the query sequence are absent from the consensus profile.  2% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle14');
          addPlot('toggle14', 'files/json/protein_data.fasta_14_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle14');">94%&nbsp;conserved; 4%&nbsp;extra; 2%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle14');
                   addPlot('toggle14', 'files/json/protein_data.fasta_14_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '660', '');                                   addPlot('toggle14', 'files/json/protein_data.fasta_14_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '622.7250344931327', '-0.45170261274150075'); addPlot('toggle14', 'files/json/protein_data.fasta_14_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');                                                       addPlot('toggle14', 'files/json/protein_data.fasta_14_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle14row">
    <div id="toggle14" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>20</td>
  <td data-score="100"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:85.0%;"></div></div></td>
  <td title="Definition" class="seq_definition">PB11933-RA  protein AED:1 QI:0|0|0|0|0|0|8|0|845</td>
  <td title="No. Hits">500</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 879 to 1340 amino-acids long. The query sequence is 845 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too short." onclick="showDiv(this, 'toggle20');
          addPlot('toggle20', 'files/json/protein_data.fasta_20_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '845', '');
    AddExplanation(this, 'toggle20');">845&nbsp;[879,&nbsp;1340]</td>
        <td title="Length Rank" class="success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  845 amino-acids long. BLAST identified 500 hit sequences with lengths from 116 to 3500 amino-acids (median: 1001; mean: 889). 186 of these hit sequences (i.e. 37%) are longer than the query sequence." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle20'); AddExplanation(this,'toggle20');">37%</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The distribution of start and/or end-coordinates of HSPs are multi-modal. To detect potential problems we performed a linear regression (with coordinates weighted inversely proportionally to  hit strength). The resulting slope is 0.1." data-conclusion="This slope is  outside our empirically calculated thresholds (0.4 and 1.2). There is no evidence that the query contains sequence from multiple genes." onclick="showDiv(this, 'toggle20');
          addPlot('toggle20', 'files/json/protein_data.fasta_20_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '771.6667790328205', '0.10236466311164398');
          addPlot('toggle20', 'files/json/protein_data.fasta_20_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');
    AddExplanation(this, 'toggle20');">0.1</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle20'); AddExplanation(this,'toggle20');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 96% amino-acid residues present in the consensus model. 0% of residues in the query sequence are absent from the consensus profile.  11% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle20');
          addPlot('toggle20', 'files/json/protein_data.fasta_20_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle20');">96%&nbsp;conserved; 0%&nbsp;extra; 11%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle20');
                   addPlot('toggle20', 'files/json/protein_data.fasta_20_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '845', '');                                   addPlot('toggle20', 'files/json/protein_data.fasta_20_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '771.6667790328205', '0.10236466311164398'); addPlot('toggle20', 'files/json/protein_data.fasta_20_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');                                                       addPlot('toggle20', 'files/json/protein_data.fasta_20_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle20row">
    <div id="toggle20" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>3</td>
  <td data-score="100"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:85.0%;"></div></div></td>
  <td title="Definition" class="seq_definition">GB10056-PA </td>
  <td title="No. Hits">499</td>
      <td title="Length Cluster" class="success my-btn-success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 280 to 399 amino-acids long. The query sequence is 339 amino-acids long and is thus inside the most dense length-cluster of BLAST hits." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle3');
          addPlot('toggle3', 'files/json/protein_data.fasta_3_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '339', '');
    AddExplanation(this, 'toggle3');">339&nbsp;[280,&nbsp;399]</td>
        <td title="Length Rank" class="success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  339 amino-acids long. BLAST identified 499 hit sequences with lengths from 125 to 2077 amino-acids (median: 323; mean: 309). 175 of these hit sequences (i.e. 35%) are shorter than the query sequence." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle3'); AddExplanation(this,'toggle3');">35%</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The distribution of start and/or end-coordinates of HSPs are multi-modal. To detect potential problems we performed a linear regression (with coordinates weighted inversely proportionally to  hit strength). The resulting slope is -0.6." data-conclusion="This slope is  outside our empirically calculated thresholds (0.4 and 1.2). There is no evidence that the query contains sequence from multiple genes." onclick="showDiv(this, 'toggle3');
          addPlot('toggle3', 'files/json/protein_data.fasta_3_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '316.72918962900036', '-0.6192804598100656');
          addPlot('toggle3', 'files/json/protein_data.fasta_3_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '499', '');
    AddExplanation(this, 'toggle3');">-0.6</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle3'); AddExplanation(this,'toggle3');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 100% amino-acid residues present in the consensus model. 1% of residues in the query sequence are absent from the consensus profile.  1% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle3');
          addPlot('toggle3', 'files/json/protein_data.fasta_3_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle3');">100%&nbsp;conserved; 1%&nbsp;extra; 1%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle3');
                   addPlot('toggle3', 'files/json/protein_data.fasta_3_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '339', '');                                   addPlot('toggle3', 'files/json/protein_data.fasta_3_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '316.72918962900036', '-0.6192804598100656'); addPlot('toggle3', 'files/json/protein_data.fasta_3_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '499', '');                                                       addPlot('toggle3', 'files/json/protein_data.fasta_3_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle3row">
    <div id="toggle3" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>21</td>
  <td data-score="100"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:85.0%;"></div></div></td>
  <td title="Definition" class="seq_definition">PB13750-RA  protein AED:1 QI:198|0|0|0|0|0|12|764|974</td>
  <td title="No. Hits">500</td>
      <td title="Length Cluster" class="success my-btn-success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 837 to 1007 amino-acids long. The query sequence is 974 amino-acids long and is thus inside the most dense length-cluster of BLAST hits." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle21');
          addPlot('toggle21', 'files/json/protein_data.fasta_21_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '974', '');
    AddExplanation(this, 'toggle21');">974&nbsp;[837,&nbsp;1007]</td>
        <td title="Length Rank" class="danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  974 amino-acids long. BLAST identified 500 hit sequences with lengths from 101 to 2225 amino-acids (median: 932; mean: 925). 96 of these hit sequences (i.e. 19%) are shorter than the query sequence." data-conclusion="The sequence may be too long." onclick="showDiv(this, 'toggle21'); AddExplanation(this,'toggle21');">19%&nbsp;(too&nbsp;long)</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The start coordinates and the end coordinates of HSPs are unimodally distributed." data-conclusion="This suggest that the query sequence represents a single gene." onclick="showDiv(this, 'toggle21');
          addPlot('toggle21', 'files/json/protein_data.fasta_21_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', '');
          addPlot('toggle21', 'files/json/protein_data.fasta_21_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');
    AddExplanation(this, 'toggle21');">0.0</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle21'); AddExplanation(this,'toggle21');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 88% amino-acid residues present in the consensus model. 0% of residues in the query sequence are absent from the consensus profile.  7% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle21');
          addPlot('toggle21', 'files/json/protein_data.fasta_21_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle21');">88%&nbsp;conserved; 0%&nbsp;extra; 7%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle21');
                   addPlot('toggle21', 'files/json/protein_data.fasta_21_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '974', '');                                   addPlot('toggle21', 'files/json/protein_data.fasta_21_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', ''); addPlot('toggle21', 'files/json/protein_data.fasta_21_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');                                                       addPlot('toggle21', 'files/json/protein_data.fasta_21_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle21row">
    <div id="toggle21" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>5</td>
  <td data-score="75"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:63.75%;"></div></div></td>
  <td title="Definition" class="seq_definition">GB10063-PA </td>
  <td title="No. Hits">498</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 581 to 606 amino-acids long. The query sequence is 615 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too long." onclick="showDiv(this, 'toggle5');
          addPlot('toggle5', 'files/json/protein_data.fasta_5_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '615', '');
    AddExplanation(this, 'toggle5');">615&nbsp;[581,&nbsp;606]</td>
        <td title="Length Rank" class="danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  615 amino-acids long. BLAST identified 498 hit sequences with lengths from 460 to 874 amino-acids (median: 597; mean: 601). 64 of these hit sequences (i.e. 13%) are shorter than the query sequence." data-conclusion="The sequence may be too long." onclick="showDiv(this, 'toggle5'); AddExplanation(this,'toggle5');">13%&nbsp;(too&nbsp;long)</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The start coordinates and the end coordinates of HSPs are unimodally distributed." data-conclusion="This suggest that the query sequence represents a single gene." onclick="showDiv(this, 'toggle5');
          addPlot('toggle5', 'files/json/protein_data.fasta_5_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', '');
          addPlot('toggle5', 'files/json/protein_data.fasta_5_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '498', '');
    AddExplanation(this, 'toggle5');">0.0</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 0.500." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle5'); AddExplanation(this,'toggle5');">0.5</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 100% amino-acid residues present in the consensus model. 0% of residues in the query sequence are absent from the consensus profile.  0% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle5');
          addPlot('toggle5', 'files/json/protein_data.fasta_5_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle5');">100%&nbsp;conserved; 0%&nbsp;extra; 0%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle5');
                   addPlot('toggle5', 'files/json/protein_data.fasta_5_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '615', '');                                   addPlot('toggle5', 'files/json/protein_data.fasta_5_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', ''); addPlot('toggle5', 'files/json/protein_data.fasta_5_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '498', '');                                                       addPlot('toggle5', 'files/json/protein_data.fasta_5_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle5row">
    <div id="toggle5" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>16</td>
  <td data-score="25"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:21.25%;"></div></div></td>
  <td title="Definition" class="seq_definition">Or13a-like-1  protein AED:1 QI:0|0|0|0|0|0|23|0|774</td>
  <td title="No. Hits">398</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 357 to 422 amino-acids long. The query sequence is 774 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too long." onclick="showDiv(this, 'toggle16');
          addPlot('toggle16', 'files/json/protein_data.fasta_16_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '774', '');
    AddExplanation(this, 'toggle16');">774&nbsp;[357,&nbsp;422]</td>
        <td title="Length Rank" class="danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  774 amino-acids long. BLAST identified 398 hit sequences with lengths from 54 to 1706 amino-acids (median: 389; mean: 346). 7 of these hit sequences (i.e. 2%) are shorter than the query sequence." data-conclusion="The sequence may be too long." onclick="showDiv(this, 'toggle16'); AddExplanation(this,'toggle16');">2%&nbsp;(too&nbsp;long)</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The distribution of start and/or end-coordinates of HSPs are multi-modal. To detect potential problems we performed a linear regression (with coordinates weighted inversely proportionally to  hit strength). The resulting slope is -0.2." data-conclusion="This slope is  outside our empirically calculated thresholds (0.4 and 1.2). There is no evidence that the query contains sequence from multiple genes." onclick="showDiv(this, 'toggle16');
          addPlot('toggle16', 'files/json/protein_data.fasta_16_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '759.1456041567415', '-0.16523438625139966');
          addPlot('toggle16', 'files/json/protein_data.fasta_16_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '398', '');
    AddExplanation(this, 'toggle16');">-0.2</td>
        <td title="Duplication" class="danger" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 0.002 (average = 1.31)." data-conclusion="The null hypothesis is rejected - thus a region of the query sequence is likely repeated more than once." onclick="showDiv(this, 'toggle16'); AddExplanation(this,'toggle16');">0.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="danger my-btn-danger" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 90% amino-acid residues present in the consensus model. 43% of residues in the query sequence are absent from the consensus profile.  8% of residues in the consensus profile are absent from the query sequence." data-conclusion="These results suggest that there may be some problems with the query sequence. The query sequence has a high percentage (43%) of extra residues absent from the statistical profile (the cut-off is 20%)." onclick="showDiv(this, 'toggle16');
          addPlot('toggle16', 'files/json/protein_data.fasta_16_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle16');">90%&nbsp;conserved; 43%&nbsp;extra; 8%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle16');
                   addPlot('toggle16', 'files/json/protein_data.fasta_16_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '774', '');                                   addPlot('toggle16', 'files/json/protein_data.fasta_16_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '759.1456041567415', '-0.16523438625139966'); addPlot('toggle16', 'files/json/protein_data.fasta_16_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '398', '');                                                       addPlot('toggle16', 'files/json/protein_data.fasta_16_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle16row">
    <div id="toggle16" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>12</td>
  <td data-score="100"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:85.0%;"></div></div></td>
  <td title="Definition" class="seq_definition">SI2.2.0_02565  locus=Si_gnF.scaffold10535[2342913..2350141].pep_2 quality=99.83 (SI2.2.3_proteins.fa_35)</td>
  <td title="No. Hits">498</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 113 to 421 amino-acids long. The query sequence is 1761 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too long." onclick="showDiv(this, 'toggle12');
          addPlot('toggle12', 'files/json/protein_data.fasta_12_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '1761', '');
    AddExplanation(this, 'toggle12');">1761&nbsp;[113,&nbsp;421]</td>
        <td title="Length Rank" class="success" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  1761 amino-acids long. BLAST identified 498 hit sequences with lengths from 113 to 5266 amino-acids (median: 406; mean: 982). 144 of these hit sequences (i.e. 29%) are shorter than the query sequence." data-conclusion="There is no reason to believe there is any problem with the length of the query sequence." onclick="showDiv(this, 'toggle12'); AddExplanation(this,'toggle12');">29%</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The distribution of start and/or end-coordinates of HSPs are multi-modal. To detect potential problems we performed a linear regression (with coordinates weighted inversely proportionally to  hit strength). The resulting slope is -0.2." data-conclusion="This slope is  outside our empirically calculated thresholds (0.4 and 1.2). There is no evidence that the query contains sequence from multiple genes." onclick="showDiv(this, 'toggle12');
          addPlot('toggle12', 'files/json/protein_data.fasta_12_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '1607.4781461713521', '-0.19899043665658656');
          addPlot('toggle12', 'files/json/protein_data.fasta_12_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '498', '');
    AddExplanation(this, 'toggle12');">-0.2</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 1." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle12'); AddExplanation(this,'toggle12');">1.0</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 95% amino-acid residues present in the consensus model. 2% of residues in the query sequence are absent from the consensus profile.  4% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle12');
          addPlot('toggle12', 'files/json/protein_data.fasta_12_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle12');">95%&nbsp;conserved; 2%&nbsp;extra; 4%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle12');
                   addPlot('toggle12', 'files/json/protein_data.fasta_12_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '1761', '');                                   addPlot('toggle12', 'files/json/protein_data.fasta_12_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '1607.4781461713521', '-0.19899043665658656'); addPlot('toggle12', 'files/json/protein_data.fasta_12_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '498', '');                                                       addPlot('toggle12', 'files/json/protein_data.fasta_12_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle12row">
    <div id="toggle12" class="expanded-child"></div>
  </td>
</tr>
<tr>
  <td>23</td>
  <td data-score="75"><div class="ratings"><div class="empty-stars"></div><div class="full-stars" style="width:63.75%;"></div></div></td>
  <td title="Definition" class="seq_definition">PB20127-RA  protein AED:1 QI:0|0|0|0|0|0|8|0|679</td>
  <td title="No. Hits">500</td>
      <td title="Length Cluster" class="danger my-btn-danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths. Here, we cluster the lengths of hit sequences and compare the length of our query sequence to the most dense cluster of hit lengths. " data-explanation="The most dense length-cluster of BLAST hits includes sequences that are from 539 to 650 amino-acids long. The query sequence is 679 amino-acids long and is thus outside the most dense length-cluster of BLAST hits." data-conclusion="This suggests that the query sequence may be too long." onclick="showDiv(this, 'toggle23');
          addPlot('toggle23', 'files/json/protein_data.fasta_23_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '679', '');
    AddExplanation(this, 'toggle23');">679&nbsp;[539,&nbsp;650]</td>
        <td title="Length Rank" class="danger" data-approach="If the query sequence is well conserved and similar sequences (BLAST hits) are correct, we can expect query and hit sequences to have similar lengths." data-explanation="The query sequence is  679 amino-acids long. BLAST identified 500 hit sequences with lengths from 125 to 3211 amino-acids (median: 608; mean: 614). 49 of these hit sequences (i.e. 10%) are shorter than the query sequence." data-conclusion="The sequence may be too long." onclick="showDiv(this, 'toggle23'); AddExplanation(this,'toggle23');">10%&nbsp;(too&nbsp;long)</td>
        <td title="Gene Merge" class="success my-btn-success" data-approach="We expect the query sequence to encode a single protein-coding gene. Here, we analyse the High-scoring Segment Pairs (HSPs) identified by BLAST to determine whether the query includes sequence from two or more genes." data-explanation="The start coordinates and the end coordinates of HSPs are unimodally distributed." data-conclusion="This suggest that the query sequence represents a single gene." onclick="showDiv(this, 'toggle23');
          addPlot('toggle23', 'files/json/protein_data.fasta_23_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', '');
          addPlot('toggle23', 'files/json/protein_data.fasta_23_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');
    AddExplanation(this, 'toggle23');">0.0</td>
        <td title="Duplication" class="success" data-approach="We expect each BLAST hit to match each region of the query at most once. Here, we calculate the distribution of hit coverage against the query sequence and use the Wilcoxon test to determine if it is higher than 1." data-explanation="The Wilcoxon test produced a p-value of 0.500." data-conclusion="This suggests that the query sequence contains no erroneous duplications." onclick="showDiv(this, 'toggle23'); AddExplanation(this,'toggle23');">0.5</td>
        <td title="" class="warning"></td>
        <td title="" class="warning"></td>
        <td title="Missing/Extra sequences" class="success my-btn-success" data-approach="We expect the query sequence to be similar to the top ten BLAST hits. Here, we create a statistical consensus model of those top hits and compare the query to this model." data-explanation="The query sequence includes 98% amino-acid residues present in the consensus model. 5% of residues in the query sequence are absent from the consensus profile.  14% of residues in the consensus profile are absent from the query sequence." data-conclusion="There is no evidence based on the top 10 BLAST hits to suggest any problems with the query sequence." onclick="showDiv(this, 'toggle23');
          addPlot('toggle23', 'files/json/protein_data.fasta_23_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');
    AddExplanation(this, 'toggle23');">98%&nbsp;conserved; 5%&nbsp;extra; 14%&nbsp;missing.</td>
  
  <td>
    <button title="Show plots" name="plot_btn" class="plot_btn btn btn-default" onclick="showDiv(this, 'toggle23');
                   addPlot('toggle23', 'files/json/protein_data.fasta_23_len_clusters.json', 'bars', 'Length Cluster Validation: Distribution of BLAST hit lengths', 'Query Sequence, black;Most Dense Cluster,red;Other Hits, blue', 'Sequence Length', 'Number of Sequences', '679', '');                                   addPlot('toggle23', 'files/json/protein_data.fasta_23_match_2d.json', 'scatter', 'Gene Merge Validation: Start/end of matching hit coord. on query (1 point/hit)', '', 'Start Offset (most left hsp)', 'End Offset (most right hsp)', '', ''); addPlot('toggle23', 'files/json/protein_data.fasta_23_match.json', 'lines', 'Gene Merge Validation: Query coord covered by blast hit (1 line/hit)', '', 'Offset in Prediction', 'Hit Number', '500', '');                                                       addPlot('toggle23', 'files/json/protein_data.fasta_23_ma.json', 'align', 'Missing/Extra sequences Validation: Multiple Align. & Statistical model of hits', 'Conserved Region, Yellow', 'Offset in the Alignment', '', '12', 'Prediction, hit&nbsp;1, hit&nbsp;2, hit&nbsp;3, hit&nbsp;4, hit&nbsp;5, hit&nbsp;6, hit&nbsp;7, hit&nbsp;8, hit&nbsp;9, hit&nbsp;10, Statistical Model');          "><i class="fa fa-bar-chart-o"></i></button>
  </td>
</tr>
<tr class="tablesorter-childRow" name="plot_row" style="display:none;">
  <td colspan="14" id="toggle23row">
    <div id="toggle23" class="expanded-child"></div>
  </td>
</tr>
</tbody>
</table>
<script>
  var evaluation_div = document.getElementById('report_1');
  var content_less = "<br>Overall Query Score Evaluation:<br>25 predictions were validated, from which there were:<br>20 are good prediction(s),<br>5 are possibly weak prediction(s).<br>1 could not be evaluated due to the lack of evidence.<div class=\"clearfix\"></div><br><button type=\"button\" onClick='add_content()' class=\"btn btn-primary btn-sm\"> Show More</button>";
  var content = "<br>Overall Query Score Evaluation:<br>25 predictions were validated, from which there were:<br>20 are good prediction(s),<br>5 are possibly weak prediction(s).<br>1 could not be evaluated due to the lack of evidence.<br><br>Average running time for LengthCluster Validation: 1.297s per validation<br>Average running time for LengthRank Validation: 0.001s per validation<br>Average running time for Gene_Merge Validation: 0.501s per validation<br>Average running time for Duplication Validation: 1.109s per validation<br>Average running time for MA Validation: 1.732s per validation <div class=\"clearfix\"></div><br><button type=\"button\" onClick='evaluation_div.innerHTML=content_less' class=\"btn btn-primary btn-sm\"> Show Less</button>";
  evaluation_div.innerHTML = content_less;
  function add_content(){
    evaluation_div = document.getElementById('report_1')
    evaluation_div.innerHTML=content;
    addPlot('report_1', 'files/json/protein_data.fasta_statistics.json', 'simplebars', 'Overall evaluation', '', 'validation score', 'number of queries');
  }
</script>
    <div aria-hidden="true" aria-labelledby="myModalLabel3" class="modal" id="spinner1" role="dialog" tabindex="-1">
      <div class="modal-dialog">
        <div class="modal-content">
          <div class="modal-body text-center">
            <h2>Loading ...</h2>
            <i class="fa fa-spinner fa-5x fa-spin"></i>
          </div>
        </div>
      </div>
    </div>

    <div class="modal fade" id="alert" aria-hidden="true" aria-labelledby="myModalLabel5" role="dialog" tabindex="-1">
      <div class="modal-dialog">
        <div class="modal-content">
          <div class="modal-header">
            <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            <h4 class="modal-title">Oops, Something went wrong!</h4>
          </div>
          <div class="modal-body">
            <p>This operation is not posible. There seems to be too many queries...</p>
          </div>
          <div class="modal-footer">
            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
          </div>
        </div>
      </div>
    </div>

    <div class="modal fade" id="browseralert" aria-hidden="true" aria-labelledby="browser" role="dialog" tabindex="-1">
      <div class="modal-dialog">
        <div class="modal-content browser-alert">
          <div class="modal-header">
            <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            <h4 class="modal-title">Incompatible Browser - Please use Mozilla Firefox!</h4>
          </div>
          <div class="modal-body" id="browseralertbody">
            <p id="browseralertText">The plugin that we use to display the validation plots means that your browser is currently unsupported.</p>
            <p>Please use <strong>Mozilla FireFox</strong> to view this file.</p>
          </div>
        </div>
      </div>
    </div>

</div>

<footer >
  <div class="container center-block">
  <p class="text-muted text-center">Please cite: "Dragan M, Moghul MI, Priyam A & Wurm Y <em>(in prep)</em> GeneValidator: identify problematic gene predictions"<br/>
  Developed at <a href="http://yannick.poulet.org" target="_blank">Wurm Lab</a>, <a href="http://www.sbcs.qmul.ac.uk" target="_blank">QMUL</a> with funding by <a href="http://www.bbsrc.ac.uk/home/home.aspx" target="_blank">BBSRC</a> and <a href="https://www.google-melange.com/gsoc/homepage/google/gsoc2013" target="_blank">Google Summer of Code 2013</a><br/>
  This page was created by <a href="https://github.com/monicadragan/GeneValidator" target="_blank" >GeneValidator</a> v1.5.0</p>
  </div>
</footer>

</body>
</html>